packages/oui-dual-list/_mixins.less

```diff
@@ -0,0 +1,55 @@
+@import '../oui-animation/_mixins';
+@import '../oui-typography/_constants';
+@import './_variables';
+
+.flexbox() {
```

Comment:
We could remove those kind of mixins. Since we support IE11 and last 2 versions of each browser.
You could use flex properties directly as is.